envs/scpca-renv.yaml

```
@@ -5,6 +5,7 @@ channels:
   - defaults
 dependencies:
   - bioconductor-annotationdbi=1.60.0
+  - bioconductor-batchelor=1.14.1
```

It looks like the latest version available on bioconda is 1.14.0
```suggestion
  - bioconductor-batchelor=1.14.0
```
https://bioconda.github.io/recipes/bioconductor-batchelor/README.html?highlight=batchelor#package-package%20&#x27;bioconductor-batchelor&#x27;

---

Ahh! Yes that did the trick, thank you 🎉